torch/onnx/_internal/diagnostics/_rules.py

```diff
@@ -844,6 +864,31 @@ class _POERules(infra.RuleCollection):
     )
     """Result from FX graph analysis to reveal unsupported FX nodes."""
 
+    fx_node_explicit_type_promotion: _FxNodeExplicitTypePromotion = dataclasses.field(
+        default=_FxNodeExplicitTypePromotion.from_sarif(
+            **{
+                "id": "FXE0013",
```

Comment:
CONFLICT!!! FXE0013 has been taken. By the way, what does FXE mean? If it only means FX exporter, do we want to utilize it to further separate rules under FX exporter?

Reply:
good point, do we have any way to find this conflict? maybe a test that runs on ci?

Reply:
don't worry, it is merge conflicts 🤣